CHANGELOG.md

```diff
@@ -11,13 +11,16 @@ Recommendation: for ease of reading, use the following order:
 - Fixed
 -->
 
-## Unreleased
+## [0.225.0] - 2025-02-20
```

Comment:
Comment for readers:

Since we had a [hidden release](https://github.com/kamu-data/kamu-node/commit/94a633b804fe5223b06292ad1ede1375c8107fed), I'm deliberately upgrading the minor version